.github/workflows/maven.yaml

```diff
@@ -0,0 +1,40 @@
+name: Publish icu4j.jar/utilities.jar to GH Maven
+on:
+  release:
+    types: [created]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: icu4j ant releaseCLDR
+        run: ant -noinput releaseCLDR -f icu4j/build.xml
+      - name: deploy it
+        run: |
+          MYVERSION=$(echo ${GITHUB_REF} | cut -d/ -f3 | tr . _)
+          echo Github Ref ${GITHUB_REF} @ ${GITHUB_SHA}, version ${MYVERSION}
```

Comment:
Where does `MYVERSION` get set?  I may have forgotten it if I saw it before.  I'm curious about how it persists / propagates across GH Action workflows.

Reply:
☝️  it gets set on the prior line 19.  It's literally the git tag (ref) but with `.` changed to `_`

so if you tag `cldr-9.8` GITHUB_REF will be something like `refs/tags/cldr-9.8` (don't quote me on the refs/tags part) and this sets `MYVERSION=cldr-9_8`

(PS thanks for looking at this. This is a greenfield here, so appreciate the input)